publisher/monitor/price/monitor-price.py

```diff
@@ -75,7 +75,7 @@ async def main():
             print(e)
             print(traceback.format_exc())
 
-            slack_text = f"Error with Pontis price<!channel>"
+            slack_text = "Error with Pontis price<!channel>"
```

Comment:
I see I was wondering why previous commit was failling on formatting. this fixes 'f-string is missing placeholders' error?

Reply:
Yes